src/crc32c.ts

```diff
@@ -257,10 +257,16 @@ class CRC32C implements CRC32CValidator {
   static async fromFile(file: PathLike) {
     const crc32c = new CRC32C();
 
-    await new Promise((resolve, reject) => {
+    await new Promise<void>((resolve, reject) => {
```

Comment:
I think you will need to pull in the latest changes from main as well. I recently fixed this issue as it was causing nightly build breaks.